### DISCUSSION TITLE
Crashes in between

### DISCUSSION THREAD

**Discussion by Jani-pro** [kimlimjustin/xplorer#93] [General]
hey there never seen this type of good ui interface in file explorer but due to some reason it doesnt work with mine pc it just show the home page but when i try to open local disk d ,e...it just doesnt please look into it

**Comment by kimlimjustin** [comment]
Firstly, have you tried to reopen the app

If you reopened and still crashes, try to disable the `Extract EXE file icon and make it a preview` setting on the Settings page. (This has been addressed as Common Problems [here](https://xplorer.vercel.app/docs/common_problems/)

If the above solution doesn't work. could you please type `Ctrl + Shift + I` and direct to `Console` (just like chrome's webtool) and send a screenshot of what happening?
(I might reply you in eight hours since It's quite late here)

Thanks!
